airflow/contrib/hooks/salesforce_hook.py

```diff
@@ -18,127 +18,109 @@
 # under the License.
 #
 """
-This module contains a Salesforce Hook
-which allows you to connect to your Salesforce instance,
-retrieve data from it, and write that data to a file
-for other uses.
+This module contains a Salesforce Hook which allows you to connect to your Salesforce instance,
+retrieve data from it, and write that data to a file for other uses.
 
-NOTE:   this hook also relies on the simple_salesforce package:
-        https://github.com/simple-salesforce/simple-salesforce
+.. note:: this hook also relies on the simple_salesforce package:
+      https://github.com/simple-salesforce/simple-salesforce
 """
-from simple_salesforce import Salesforce
-from airflow.hooks.base_hook import BaseHook
-
-import json
+import time
 
 import pandas as pd
-import time
+from simple_salesforce import Salesforce
 
+from airflow.hooks.base_hook import BaseHook
 from airflow.utils.log.logging_mixin import LoggingMixin
 
 
 class SalesforceHook(BaseHook):
-    def __init__(
-            self,
-            conn_id,
-            *args,
-            **kwargs
-    ):
+    def __init__(self, conn_id):
```

Comment:
Could you add the `@apply_defaults` decorator here?

Reply:
Done. :)

Reply:
Wait.. Isn't the `@apply_defaults` only necessary for operators?

> def apply_defaults(func):
    """
    Function decorator that Looks for an argument named "default_args", and
    fills the unspecified arguments from it.

Reply:
I reverted it. Please let me know if I am wrong and I will add it back in.

Reply:
this is only for operators. it makes it easier to create DAGs